src/leaflike/bookmarks/db.clj

```diff
@@ -2,14 +2,23 @@
   (:require [clojure.java.jdbc :as jdbc]
             [leaflike.config :refer [db-spec]]
             [honeysql.core :as sql]
-            [honeysql.helpers :as helpers]))
+            [honeysql.helpers :as helpers]
+            [honeysql.types :as types]))
+
+(defn format-tags
+  [params]
+  ;; check if tag exists
+  ;; check if tags is nil?
+  (if (nil? (get params :tags))
+    params
+    (update params :tags #(types/array %))))
 
 (defn create
   [params]
```

Comment:
Let's destructure `params` here like the rest of the functions in this namespace. Right now, it's not very clear what this function expects the `params` map to be.